### DISCUSSION TITLE
Bad translations

### DISCUSSION THREAD

**Discussion by MarioParrilla** [community/community#39085] [Mobile]
The translations on the app theme in Spanish are bad, should be dark like Oscuro and light like Claro
![image](https://user-images.githubusercontent.com/8973790/201696272-fbe87f39-6e17-4fd0-9380-cbc9c8265d7c.png)

**Comment by aconsuegra** [comment]
Hi @MarioParrilla, thanks for your feedback! I'm going to create a tracking issue so we can fix this in our next release.
